migratedb.d/04.sql

```diff
@@ -777,13 +779,32 @@ BEGIN
     -- Set new permissions                                                   --
     ---------------------------------------------------------------------------
 
-    CREATE ROLE base;
+    -- Temporary function for creating roles if they do not already exist.
+    CREATE FUNCTION create_role_if_not_exists(role_name NAME) RETURNS void AS
+    $created$
+    BEGIN
```

Comment:
Not a fan of putting stuff on a separate line without specific reason.

I prefer it to look like this:
```suggestion
    -- Temporary function for creating roles if they do not already exist.
    CREATE FUNCTION create_role_if_not_exists(role_name NAME) RETURNS void AS $created$
    BEGIN
```